lib/CXGN/Onto.pm

```diff
@@ -157,21 +157,21 @@ sub store_composed_term {
           accession => sprintf("%07d",$accession)
       });
 
-      my $parent_term= $schema->resultset("Cv::Cvterm")->find(
-        { cv_id  =>$cv->cv_id(),
+      my $parent_term= $schema->resultset("Cv::Cvterm")->create_with(
```

Comment:
shouldn't this be something like find_or_create_with? You want to create this only once?